src/server/game/Entities/Item/Item.h

```diff
@@ -124,6 +124,39 @@ struct ItemDynamicFieldGems
 };
 #pragma pack(pop)
 
+int32 const ItemTransmogrificationSlots[MAX_INVTYPE] =
```

Comment:
This change is bad, all you had to do here was write a declaration - `extern int32 const ItemTransmogrificationSlots[MAX_INVTYPE]`